ulscrape/items.py

```diff
@@ -5,10 +5,11 @@
 # See documentation in:
 # http://doc.scrapy.org/en/latest/topics/items.html
 
-import scrapy
+from scrapy import Item, Field
 
 
-class UlscrapeItem(scrapy.Item):
-    # define the fields for your item here like:
-    # name = scrapy.Field()
-    pass
+class UlscrapeItem(Item):
```

Comment:
It is good idea to name "item" as your core business domain,... Scraping cars = car, scraping hotels = hotel,... Scraping documents = document,...

Can I recommend that you rename / rewrite this class like this:

```python
class Document(Item):
  scraped_at = Field()
  #...
```